rt/rs/extensions/reactor/src/main/java/org/apache/cxf/jaxrs/reactor/server/ReactorCustomizer.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.cxf.jaxrs.reactor.server;
+
+import java.util.Collections;
+
+import org.apache.cxf.jaxrs.JAXRSServerFactoryBean;
+import org.apache.cxf.jaxrs.ext.JAXRSServerFactoryCustomizationExtension;
+import org.apache.cxf.jaxrs.provider.StreamingResponseProvider;
+
+public class ReactorCustomizer implements JAXRSServerFactoryCustomizationExtension {
+    @Override
+    public void customize(JAXRSServerFactoryBean bean) {
+        Boolean useStreamingSubscriber = (Boolean)bean.getProperties(true)
+                .getOrDefault("useStreamingSubscriber", false);
+        ReactorInvoker invoker = new ReactorInvoker();
+        invoker.setUseStreamingSubscriberIfPossible(useStreamingSubscriber);
+        bean.setInvoker(invoker);
+        StreamingResponseProvider<Object> streamProvider = new StreamingResponseProvider<>();
+        streamProvider.setProduceMediaTypes(Collections.singletonList("application/json"));
```

Comment:
Minor, may be `MediaType.APPLICATION_JSON` instead of `application/json`?

Reply:
Agreed.  Actually, after I pulled this out I decided I really need a base class for these extensions.    So I'm going to take care of that before merging.

Reply:
Sounds good, thanks!